src/main/java/bitbot/Application.java

```diff
@@ -0,0 +1,162 @@
+package bitbot;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Scanner;
+
+public class Application {
+    private static final String PATH_TO_FILE = "./data/Bitbot.txt";
+
+    public static void run() throws FileNotFoundException{
+
+        ArrayList<Task> arrayList = Storage.readTasksFromFile(PATH_TO_FILE);
+
+        String inputData;
+
+        //printing out the intro
+        System.out.println(UI.intro);
+        //creating a scanner
+        Scanner sc = new Scanner(System.in);
+
+        while (true) {
+            StringBuilder from = new StringBuilder();
+            StringBuilder to = new StringBuilder();
+            StringBuilder by = new StringBuilder();
+            String keyWord = null;
+            String textPart = null;
+            StringBuilder sb = new StringBuilder();
+
+            int numberPart = -1;
+            // to store the index when /from is present so that
+            // it aids me in getting the description
+            int indexFrom = -1;
+            // to store the index when /by is present so that
+            // it aids be in getting the description
+            int indexBy = -1;
+
+            // reading each line
+            inputData = sc.nextLine();
+
+            // this is to help check if the size is more than 1 or not
+            // so that I can use the correct term (either "task" or "tasks")
+            String task = "";
+            if (arrayList.size() > 0) {
+                task = "tasks";
+            } else {
+                task = "task";
+            }
+
+            try {
+                String[] partsOfInput = inputData.split(" ");
+
+                /* this is to check if there is any number in the input.
+                 * I am splitting the input into parts, and then
+                 * I am doing a check. If the last element is an integer,
+                 * I will go ahead and save it as an integer. If not,
+                 * it will be made into a string using StringBuilder.
+                 * The .trim() is used to remove the trailing spaces.
+                 */
+                if (partsOfInput.length > 0) {
+                    keyWord = partsOfInput[0];
+
+                    switch (keyWord) {
```

Comment:
I also just found out, but according to our coding standard, there should be no indentation for case clauses.